Unable to load service index for darc-pub-dotnet-runtime-488a8a35 in WASM CI legs

### Build

https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=1149050

### Build leg reported

NoWorkload-NoWebcil-Wasm.Build.Tests.WorkItemExecution

### Pull Request

https://github.com/dotnet/runtime/pull/119740

### Known issue core information

Fill out the known issue JSON section by following the [step by step documentation on how to create a known issue](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md#how-to-create-a-known-issue-step-by-step)

```json
 {
    "ErrorMessage" : "error NU1301: Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-488a8a35/nuget/v3/index.json",
    "BuildRetry": false,
    "ErrorPattern": "",
    "ExcludeConsoleLog": false
 }
 ```

 @dotnet/dnceng

<!-- DO NOT DELETE -->
<!-- For internal use only; put release notes here. -->
<!-- For guidance on writing good release notes, please see documentation here: https://dev.azure.com/dnceng/internal/_wiki/wikis/DNCEng%20Services%20Wiki/983/ReleaseNotesGuidance -->
<!-- Additionally, please specify the note category below. -->
### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements
### Release Note Description


### Additional information about the issue reported

This is occurring only for the release/8.0 and release/8.0-staging legs

--- a/src/mono/wasi/Wasi.Build.Tests/data/nuget8.config
+++ b/src/mono/wasi/Wasi.Build.Tests/data/nuget8.config
@@ -9,11 +9,6 @@
     <!-- TEST_RESTORE_SOURCES_INSERTION_LINE -->
     <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="nuget.org"  value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-
-    <!-- work around illink needing a stable version for now -->
-    <!--  Begin: Package sources from dotnet-runtime -->
-    <add key="darc-pub-dotnet-runtime-488a8a3" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-488a8a35/nuget/v3/index.json" />
-    <!--  End: Package sources from dotnet-runtime -->
   </packageSources>
     <disabledPackageSources>
     <clear />

--- a/src/mono/wasm/Wasm.Build.Tests/data/nuget8.config
+++ b/src/mono/wasm/Wasm.Build.Tests/data/nuget8.config
@@ -9,11 +9,6 @@
     <!-- TEST_RESTORE_SOURCES_INSERTION_LINE -->
     <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="nuget.org"  value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-
-    <!-- work around illink needing a stable version for now -->
-    <!--  Begin: Package sources from dotnet-runtime -->
-    <add key="darc-pub-dotnet-runtime-488a8a3" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-488a8a35/nuget/v3/index.json" />
-    <!--  End: Package sources from dotnet-runtime -->
   </packageSources>
     <disabledPackageSources>
     <clear />

